### PR TITLE
Database additions

### LINKED ISSUE #185: Add ways of getting to Sunburst location
After solving the Ing Windchamber puzzle, you can scan dash to the platform with the cannon (from either side).

Alternatively, with Space Jump Boots, you can abuse a terminal fall to get there (https://www.twitch.tv/videos/417963262). The jumps are very easy, but I think the first one is only possible from the side of the room with the portal.

### PR DESCRIPTION
**Pull request by SpaghettiToastBook**
Fixes:
- #115
- #136
- #142
- #185
- #213

Also:
- Adds missing requirements in Hydrodynamo Station to Door to Hydrodynamo Shaft → Door to Save Station B (namely, the requirements to unlock all of the locks)
- Adds lots of requirements implied by other ones (mostly for Morph Ball, Charge Beam, and Scan Visor)

### CODE REVIEW

**Comment by henriquegemignani** [comment]
Whenever changing an area, I like working towards #103. It's having less nodes with connections if they're redundant, making generic nodes in the room if needed. Also adding Morph Ball/Scan Visor requirements.

Would you be able to make these changes to Grand Windchamber? If not, I can merge as is.

File: randovania/data/json_data/prime2.json [hunk 1]
```diff
@@ -12946,6 +13088,84 @@
                                         }
                                     ]
                                 ],
+                                "Pickup (Darkburst)": [
```
Comment (henriquegemignani) [inline randovania/data/json_data/prime2.json]: Add these as paths to `Door to Transit Station`.

**Comment by SpaghettiToastBook** [comment]
> Whenever changing an area, I like working towards #103. It's having less nodes with connections if they're redundant, making generic nodes in the room if needed. Also adding Morph Ball/Scan Visor requirements.
> 
> Would you be able to make these changes to Grand Windchamber? If not, I can merge as is.

Should I add or remove nodes in Grand Windchamber?

**Comment by henriquegemignani** [comment]
Whatever you feel is sufficient for the room.
For this room, I think just having things connect to one door and the door connect to things may work well.

File: randovania/data/json_data/prime2.json (hunk 1 above)
Comment (SpaghettiToastBook) [reply randovania/data/json_data/prime2.json]: I'm planning to reorganize the connections here in the spirit of #103. I'll make sure to do this.

**Pull request merged**